libs/client-api/Cargo.toml

```diff
@@ -42,10 +42,15 @@ serde.workspace = true
 database-entity.workspace = true
 app-error = { workspace = true, features = ["tokio_error", "bincode_error"] }
 scraper = { version = "0.17.1", optional = true }
+web-sys = { version = "0.3.67", features = ["BinaryType"] }
```

Comment:
please add the deps in the [target.'cfg(target_arch = "wasm32")'.dependencies] section. Also the deps is duplicated